use super::MessageError;

pub async fn handler() -> Result<(), MessageError> {
    Ok(())
}
